number_parser_scripts/write_complete_language_data.py

```diff
@@ -138,7 +137,14 @@ def write_complete_data():
         full_supplementary_path = os.path.join(SUPPLEMENTARY_PATH, file_name)
 
         language_data = {key: {} for key in REQUIRED_NUMBERS_DATA}
-        ordered_language_data = OrderedDict((key, {}) for key in REQUIRED_NUMBERS_DATA)
+        ordered_language_data = {
+            "UNIT_NUMBERS": {},
+            "DIRECT_NUMBERS": {},
+            "TENS": {},
+            "HUNDREDS": {},
+            "BIG_POWERS_OF_TEN": {},
+        }
+        ordered_language_data = {key: {} for key in REQUIRED_NUMBERS_DATA}
```

Comment:
I think this is what @wRAR meant:

```suggestion
        ordered_language_data = {key: {} for key in REQUIRED_NUMBERS_DATA}
```